src/sicks3000.cc

```diff
@@ -123,8 +123,13 @@ int SickS3000::Close()
     // Read controller messages.
     if (serial->ReadPort(cReadBuffer, &read_bytes, 1999) < 0)
```

Comment:
Is this return value an `errno`? Could use `strerror()` to print a user friendly version if that's the case: http://beej.us/guide/bgnet/output/html/multipage/perrorman.html